notebooks/README.md

```diff
@@ -261,6 +261,11 @@ demonstrates using DeepZ to compute certified robustness for neural networks.
   <img src="../utils/data/images/zonotope_picture.png?raw=true" width="200" title="poisoning">
 </p>
 
+## Certified Training
```

Comment:
Would it be possible to add here the image of the notebook showing zonotope shape as graphical abstract?

Reply:
Added new image to readme and notebook